lib/spack/spack/build_systems/autotools.py

```diff
@@ -175,11 +198,14 @@ def _run_default_function(self):
             tty.msg('Skipping default sanity checks [method `check` not implemented]')  # NOQA: ignore=E501
 
     def check(self):
```

Comment:
I wouldn't change the `check` method here. If you need some custom behavior for `Blitz++` it's better to code it in the package.